app/assets/stylesheets/mixins/_forum_post.scss

```diff
@@ -0,0 +1,38 @@
+@mixin forum_post {
+  .post {
+    // In case the previous post was too short.
+    clear: both;
+
+    // Spacing between posts.
+    margin-bottom: 1em;
+
+    .user {
+      margin-right: 1em;
+      text-align: right;
+      width: 7em;
+      float: left;
+    }
+
+    .contents {
+      padding: 20px;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, line-height, overflow, padding